docs/reference/index.md

```diff
@@ -1,23 +1,26 @@
 # Schema reference
 
+The Infrastructure Project Level Data Specification is described using JSON Schema 0.4.
+
+The [schema file is available here](../../../../_static/project-level/project-schema.json) and can be explored using the [schema browser](schema.md).
+
+Full schema and codelist [reference documentation is also available](reference.md).
+
+The [packaging data section](project_package.md) describes how to package OC4IDS data for publication. 
```

Comment:
"packaging" is not a common concept, so some context would be good – something like "To provide easier access to bulk data, the data about individual projects can be combined into a 'package' (link)." Alternately, we should reframe the normative language from the package page about how data must be published.

Reply:
I reframed the normative language from the package page in https://github.com/open-contracting/infrastructure/pull/89/commits/381b67bbdee4987956315a5b528d5b5f8538634e